docs/source/intro/index.rst

```diff
@@ -1,12 +1,31 @@
-============
-Introduction
-============
```

Comment:
I would keep this, because now it says "What is AiiDA" in the top navigation bar which is weird I find. I would prefer it to be "Introduction" which fits better with the rest of the subsections.